libafl/src/generators/nautilus.rs

```diff
@@ -85,7 +86,15 @@ impl NautilusContext {
 
     /// Create a new [`NautilusContext`] from a file
     #[must_use]
-    pub fn from_file<P: AsRef<Path>>(tree_depth: usize, grammar_file: P) -> Self {
+    pub fn from_file(tree_depth: usize, grammar_file: PathBuf) -> Self {
```

Comment:
Why not keep `AsRef<Path>` here? It's a much cleaner API.

Reply:
fair enough